global-api/importer/edgar/citycelloverlapedgar_importer.py

```diff
@@ -0,0 +1,97 @@
+# import EDGAR city data into database
+# >> python citycelloverlapedgar_importer.py --database-uri DB_URI
+# author: L. Gloege
+# created: 2023-09-28
+
+import argparse
+from datetime import datetime
+import os
+import pandas as pd
+from sqlalchemy import create_engine, MetaData, Table
+from sqlalchemy.orm import sessionmaker
+from tqdm import tqdm
+from utils import (
+    all_locodes_and_geometries,
+    area_of_polygon,
+    bounds_from_polygon,
+    get_edgar_grid_coords_and_wkt,
+    insert_record,
+    load_wkt,
+    uuid_generate_v3,
+)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--database_uri",
+        help="database URI (e.g. postgresql://ccglobal:@localhost/ccglobal)",
+        default=os.environ.get("DB_URI"),
+    )
+    args = parser.parse_args()
+
+    engine = create_engine(args.database_uri)
+    metadata_obj = MetaData()
+    Session = sessionmaker(bind=engine)
+    session = Session()
+
+    list_grid_coords = get_edgar_grid_coords_and_wkt(session)
+    df_grid = (
+        pd.DataFrame(list_grid_coords)
+        .rename(columns={"id": "cell_id"})
+        .astype({"cell_id": str})
+    )
+
+    table = Table("CityCellOverlapEdgar", metadata_obj, autoload_with=engine)
+
+    lon_res = 0.1
+    lat_res = 0.1
```

Comment:
These should maybe be defined as constants at the top. So they are easy to adjust if we need to for e.g. another data source that uses a similar structure.

Reply:
Thanks, I'll define them at the top, they are constants for the EDGAR, but this will not be the case for telescoping grids. 

Another approach is to not hard code the values, but use the discrete difference along the lat and lon vectors. We could use [decimal](https://docs.python.org/3/library/decimal.html) to avoid floating point arithmetic error. This would allow us to write a general function that works with most grids. I decided against this for now because the resolution is constant for EDGAR so hard-coding was the simpler and quicker approach.

Reply:
Agreed, let's keep the constants but just make sure they are easy to find and reused between the scripts for EDGAR.